### ISSUE TITLE
Electron problem with MacOS Catalina

### ISSUE THREAD

**Issue by archansel** [aws-samples/amazon-chime-sdk-classroom-demo#33]
MacOS Catalina prevent the electron app to run. Please fix the configuration so we can run it in MacOS or at least give us some pointers (what should be done) to make it run in MacOS. Thanks

**Comment by p-foucht** [comment]
Hi @archansel 

I believe the root cause is related to Catalina's unzipping tool. The default tool (Archive Utility) seems to incorrectly unzip, changing the executable to a text file and a few other changes that seem to break opening the application.

I was able to open the application on Catalina by...

1. Downloading an alternative zipping tool (I used "The Unarchiver") 
2. Right click on the zip file > Open with > The Unarchiver
3. Try opening the application. If you get some kind of "unidentified developer" message, you'll need to go into your Security & Privacy settings and allow the application to be opened

[See this thread for more information](https://superuser.com/questions/1345755/how-to-fix-the-application-cant-be-opened-on-mac)
